sdk/management/samples/src/test/resources/session-records/testAutoscaleSettingsBasedOnPerformanceOrSchedule.json

```diff
@@ -1,483 +1 @@
Uri": "http://localhost:1234/subscriptions/00000000-0000-0000-0000-000000000000/operationresults/eyJqb2JJZCI6IlJFU09VUkNFR1JPVVBERUxFVElPTkpPQi1NWVJFU09VUkNFR1JPVVAwNjQzNzg4NEUwMy1TT1VUSENFTlRSQUxVUyIsImpvYkxvY2F0aW9uIjoic291dGhjZW50cmFsdXMifQ?api-version=2019-08-01", "Headers": {"User-Agent": "Azure-SDK-For-Java/null OS:Windows 10/10.0 MacAddressHash:4c4be74f6c8257dbbe008416a4d411a9aabf191a80fa2bd61b630c829c3c3edd Java:11.0.4 (ResourceManagementClient, 2019-08-01)"}, "Response": {"date": "Tue, 24 Dec 2019 09:26:25 GMT", "content-length": "0", "expires": "-1", "retry-after": "0", "x-ms-ratelimit-remaining-subscription-reads": "11989", "StatusCode": "202", "pragma": "no-cache", "strict-transport-security": "max-age=31536000; includeSubDomains", "x-ms-correlation-request-id": "da234f0d-104d-4f58-a600-265c6785bccd", "x-content-type-options": "nosniff", "x-ms-routing-request-id": "SOUTHEASTASIA:20191224T092626Z:da234f0d-104d-4f58-a600-265c6785bccd", "location": "http://localhost:1234/subscriptions/00000000-0000-0000-0000-000000000000/operationresults/eyJqb2JJZCI6IlJFU09VUkNFR1JPVVBERUxFVElPTkpPQi1NWVJFU09VUkNFR1JPVVAwNjQzNzg4NEUwMy1TT1VUSENFTlRSQUxVUyIsImpvYkxvY2F0aW9uIjoic291dGhjZW50cmFsdXMifQ?api-version=2019-08-01", "cache-control": "no-cache", "x-ms-request-id": "da234f0d-104d-4f58-a600-265c6785bccd", "Body": ""}}, {"Method": "GET", "Uri": "http://localhost:1234/subscriptions/00000000-0000-0000-0000-000000000000/operationresults/eyJqb2JJZCI6IlJFU09VUkNFR1JPVVBERUxFVElPTkpPQi1NWVJFU09VUkNFR1JPVVAwNjQzNzg4NEUwMy1TT1VUSENFTlRSQUxVUyIsImpvYkxvY2F0aW9uIjoic291dGhjZW50cmFsdXMifQ?api-version=2019-08-01", "Headers": {"User-Agent": "Azure-SDK-For-Java/null OS:Windows 10/10.0 MacAddressHash:4c4be74f6c8257dbbe008416a4d411a9aabf191a80fa2bd61b630c829c3c3edd Java:11.0.4 (ResourceManagementClient, 2019-08-01)"}, "Response": {"date": "Tue, 24 Dec 2019 09:26:40 GMT", "content-length": "0", "expires": "-1", "retry-after": "0", "x-ms-ratelimit-remaining-subscription-reads": "11988", "StatusCode": "200", "pragma": "no-cache", "strict-transport-security": "max-age=31536000; includeSubDomains", "x-ms-correlation-request-id": "07ce96e5-e0cd-42bb-9117-bf3eb6bb1d14", "x-content-type-options": "nosniff", "x-ms-routing-request-id": "SOUTHEASTASIA:20191224T092641Z:07ce96e5-e0cd-42bb-9117-bf3eb6bb1d14", "cache-control": "no-cache", "x-ms-request-id": "07ce96e5-e0cd-42bb-9117-bf3eb6bb1d14", "Body": ""}}], "variables": ["mytestscalewebapp12506547e6", "autoscalename18b809413d995", "myresourcegroup06437884e03", "598a11e3-f18f-4bee-a1e9-4d16c6412c8f", "a38b9af6-7e25-4984-802a-575ea99a3983", "6e5df2f8039a4618acfd1c479c418124", "16e64eda-de7f-4d3e-8872-a3de3e7b5853", "cf4938c8-0ebf-4e30-91e4-d7f806880f57", "8da0c72f-1fb5-4c71-a814-19c591116e1f", "26cb10a4-8dbb-4688-97b8-5d8266e5a53b", "ebf4a399-5e43-418a-a415-89c7c2dd369f", "16aafaa0-c437-4f3c-bc5e-d973d763ba54", "572b982a-5a5c-4486-94bb-c5840567e629", "701dc744-068c-4b70-9213-ebed3f90aeb4", "6de3bb36-20b5-448c-9538-4041e5b992e4"]}
```

Comment:
just curious, why remove line ends, which makes readability good

Reply:
It is my python script's fault. Just trim for the poll in order for testing. The commit could be revert.

Reply:
Revert done.